python/test/fuzzing_test.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Python GLOME fuzz main.
+
+This module that implement some easy fuzz testing for pyglome. 
+"""
+
+from hypothesis import given, settings, strategies, example
+from cryptography.hazmat.primitives.asymmetric import x25519
+
+import pyglome
+
+
+def _glome(private_bytes, public_bytes, msg, tag, counter, min_tag_len):
+    """Calls basic utilities of glome class, accepts only documented exceptions."""
+    try:
+        private_key = x25519.X25519PrivateKey.from_private_bytes(private_bytes)
+        peer_key = x25519.X25519PublicKey.from_public_bytes(public_bytes)
+    except (ValueError, pyglome.ExchangeError):
+        return 
+
+    try:
+        tag_manager = pyglome.Glome(peer_key, private_key, min_tag_len)
+    except ValueError:
+        return
+
+    tag_manager.user_keys
+    tag_manager.peer_key
```

Comment:
Are you doing this for the side effects? If so, please add a comment.

Reply:
Added